repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneRequestHandler.scala

```diff
@@ -110,6 +110,10 @@ class KeystoneRequestHandler(identityServiceUri: String, akkaServiceClient: Akka
         val roles = (json \ "access" \ "user" \ "roles").as[JsArray].value
           .map(jsRole => Role((jsRole \ "name").as[String], (jsRole \ "tenantId").asOpt[String]))
           .toVector
+        val authenticatedBy = (json \ "access" \ "token" \ "RAX-AUTH:authenticatedBy").asOpt[JsArray]
+          .map(_.value)
+          .map(_.map(_.as[String]))
+          .map(_.toVector)
```

Comment:
Why keep it inside an Option instead of just letting it be an empty list?  (Aside from the fact that I doubt this would ever be an empty list, but it's easy enough to code for the possibility.)  Also, this can totally be a List now instead of a Vector.